allure-support/src/main/kotlin/com/kaspersky/components/alluresupport/interceptors/testrun/TestRunStateHolder.kt

```diff
@@ -0,0 +1,32 @@
+package com.kaspersky.components.alluresupport.interceptors.testrun
+
+import java.io.File
+
+/**
+ * Used to store shared state used by multiple interceptors
+ */
+class TestRunStateHolder {
+    private val _attachedVideos = mutableListOf<AttachedVideo>()
+    val attachedVideos: List<AttachedVideo>
```

Comment:
why not?
```kotlin
val attachedVideos: List<AttachedVideo> = _attachedVideos
```

Reply:
A matter of taste

Reply:
Will not the new list be instantiated on each call following your approach?